clients/pkg/promtail/targets/lokipush/pushtarget.go

```diff
@@ -170,6 +181,40 @@ func (t *PushTarget) handle(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusNoContent)
 }
 
+// handlePlaintext handles newline delimited input such as plaintext or NDJSON.
+func (t *PushTarget) handlePlaintext(w http.ResponseWriter, r *http.Request) {
```

Comment:
Do you think we can add a test to verify this all works correctly in this function ?

Reply:
Added a test 👍